spec/classes/init_spec.rb

```diff
@@ -23,6 +23,14 @@
           it { is_expected.to create_service('systemd-networkd').with_ensure('running') }
           it { is_expected.to create_service('systemd-networkd').with_enable(true) }
         end
+        context 'when enabling timesyncd' do
```

Comment:
Maybe extend the existing test where both resolved and networkd are enabled? Or at least add the ```is_expected.to_not``` to the plain test?

Reply:
I meant to add ```is_expected.not_to``` on line 13 as well.

Reply:
ups. added that as well